squape/settings.py

```diff
@@ -57,7 +57,7 @@ def logScreenshotOnPass(enabled: bool = True) -> None:
 
     Args:
         enabled (bool): A boolean value indicating whether to enable logging
-        of screenshots on PASS test result. Defaulting to True.
+            of screenshots on PASS test result. Defaulting to True.
```

Comment:
Will those changes be present in the sources too?

Reply:
I don't understand the question.